dasharo-compatibility/dcu.robot

```diff
@@ -13,6 +13,7 @@ Resource            ../variables.robot
 Resource            ../keywords.robot
 Resource            ../keys.robot
 Resource            ../pikvm-rest-api/pikvm_comm.robot
+Resource            ../lib/options/dcu.robot
```

Comment:
@filipleple Why do we need to include options lib?
This should be included automatically, based on the platform config.

Reply:
https://github.com/Dasharo/open-source-firmware-validation/pull/543/commits/74692a0053682dfeb92db24d60391ce352f1d17b#diff-7ee5e21e59cd627383812396fcf45a6e7ecb3e2f096ab5ae5203a0c6bd75989fL16-R16